app/models/read_by_users.rb

```diff
@@ -1,9 +0,0 @@
-class ReadByUsers
```

Comment:
Yes. Before I used that ReadByUsers object only here:
![image](https://github.com/famiprog/redmine_advanced_messenger/assets/4996054/cde30247-152e-4308-8c79-874f84d0f2b0)
But I saw that I can instantiate the object without having any class defined and I chose this, because no need to maintain this class in sync when adding some new fields in the read_by_users json (expl. when adding the "collapse" field)